src/SignalR/common/Http.Connections/src/Internal/HttpConnectionDispatcher.cs

```diff
@@ -51,6 +51,10 @@ internal partial class HttpConnectionDispatcher
     private readonly ILogger _logger;
     private const int _protocolVersion = 1;
 
+    private const string HeaderValueNoCache = "no-cache";
```

Comment:
I couldn't do the change to `CookieAuthenticationHandler` as a code fix, but I think we should do that too. I don't think we need the comment in `AddNoCacheHeaders` since we have this.